### DISCUSSION TITLE
Transition Stopping after Click Event

### DISCUSSION THREAD

**Discussion by saulocbarreto** [visgl/deck.gl#6951] [Q&A] [answered]
Hi!
I'm using the point cloud [example](https://github.com/visgl/deck.gl/blob/8.7-release/examples/website/point-cloud/app.js), loading a map.ply and an object.ply.
I set the rotationOrbit to rotationOrbit+360 since I don't want the map to rotate all the time so it is like a full (an invisible) rotation.

```
const updateCamera = () => {
      updateViewState(v => ({
        ...v,
        rotationOrbit: v.rotationOrbit + 360,
        transitionDuration: 2400,
        transitionInterpolator,        
        onTransitionEnd: updateCamera                
      }));      
    };
```
Now, I'm putting object.ply in the map.ply and constantly changing the object's position with a updateTrigger. 
Everything works perfectly until I click/rotate on the map.ply. After that, the object is not updated anymore until I click or click and hold again and the transition/animation stops.

Is there a way of overcoming this limitation? I'd need to allow the user to rotate the map while the position of the object keeps changing.
 Any help would be appreciated!

**Comment by Pessimistress** [comment, marked as answer]
By default `transitionInterruption` is set to `TRANSITION_EVENTS.BREAK`. You can try set it to `TRANSITION_EVENTS.IGNORE` instead, see https://deck.gl/docs/developer-guide/view-state-transitions

You can also watch the `interactionState` in `onViewStateChange` and restart the transition once dragging ends, see https://codepen.io/Pessimistress/pen/gzYydZ?editors=0010

**Reply by saulocbarreto** [reply]
That's great, it worked! Thank you!
